### PR TITLE
java-ques.

### PR DESCRIPTION
**Pull request by Premjeet1804**
## What type of PR is this? (check all applicable)

- [Premjeet ] 🚀 Added Name
- [java-questions ] ✨ Feature
- [ yes] 🌟 stared the repo
- [no ] 🐛 Grammatical Error
- [ yes] 📝 Documentation Update
- [-- ] 🚩 Other

## Description
added 3 question related to java language

## Add Link of GitHub Profile
https://github.com/Premjeet1804

### CODE REVIEW

**Comment by Premjeet1804** [comment]
i have added 3 java questions in java folder as u guided .... Thank-You

File: Languages/Java/questions.json [hunk 1]
```diff
@@ -65,12 +65,40 @@
...
+      "details": "When an array is passed to a method, a reference of the array is received by the method",
+      "questionType": "multi",
+      "trueOrFalse": false,
+      "shortAns": "The reference of the array",
+      "tag": "Java"
+    },
+    {
+      "id": 9,
+      "question": "compareTo() returns",
+      "details": "compareTo() returns an int value",
```
Comment (Tc2r1) [inline Languages/Java/questions.json:88]: please phrase the shortAns in the form of "it IS ____" 

And please give good syntax to all "question" key variables, usually ending with a question mark.

File: Languages/Java/questions.json [hunk 2]
```diff
@@ -65,12 +65,40 @@
...
+      "shortAns": " an int value",
+      "tag": "Java"
+    },
+    {
+      "id": 10,
+      "question": " Where does the system stores parameters and local variables whenever a method is invoked?",
+      "details": "The system stores parameters and local vaiables in stack in java",
+      "questionType": "multi",
+      "trueOrFalse": false,
+      "shortAns": "Stack",
```
Comment (Tc2r1) [inline Languages/Java/questions.json:100]: Please make your short answer longer and follow correct syntax.

**Review by Tc2r1 — CHANGES_REQUESTED** [review]

File: Languages/Java/questions.json (hunk 1 above)
Comment (Premjeet1804) [reply Languages/Java/questions.json:88]: ok sir

File: Languages/Java/questions.json (hunk 2 above)
Comment (Premjeet1804) [reply Languages/Java/questions.json:100]: ok sir

Comment (Premjeet1804) [reply Languages/Java/questions.json:100]: > Please make your short answer longer and follow correct syntax.

 Sir i have created changes as u have guided please view them..
Thank-You..

Comment (Premjeet1804) [reply Languages/Java/questions.json:100]: yes sir i will remember that Thank-you so much..
i learned alot contributing for you

**Comment by AndroidLion48** [comment]
Duplicate PR

**Review by AndroidLion48 — COMMENTED** [review]
This Commit has merge conflicts.

**Pull request closed without merging**